content/_data/changelogs/weekly.yml

```diff
@@ -1235,6 +1235,58 @@
       references:
         - url: https://jenkins.io/security/advisory/2017-10-11/
           title: security advisory
+- version: "2.85"
+  date: 2017-10-15
+  changes:
+    - type: rfe
+      message: >
+        Upgrade Remoting from 3.12 to 3.13.
+      pull: 3061
+      references:
+        - issue: 47132
+        - issue: 38711
+    - type: rfe
+      message: >
+        Restart agent communication related threads on both master and agents when encountering an unhandled exception, if possible, to improve stability.
+      issue: 38711
+      pull: 3017
+    - type: rfe
+      message: >
+        Improve performance by not querying queue dispatchers from the UI.
+      issue: 20046
+      pull: 3038
+    - type: rfe
+      message: >
+        Use node display name when printing "built on" message in the build log.
+      pull: 3051
+      issue: 47168
+    - type: rfe
+      message: >
+        Enable <code>cc.xml</code> to export jobs in folders recursively when accessed with a query parameter named <code>recursive</code>.
```

Comment:
Maybe makes sense to add sample link. Maybe YAGNI after https://github.com/jenkinsci/jenkins/pull/3081

Reply:
That's also what I thought.